src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,694 @@
+* Managing users and groups with the SCIM API requires Enterprise edition. For details, see [user management requirements](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles#requirements).
+* Note that managing _access grants_ is not possible with the SCIM API, you must use the **Organization and access** UI to create grants. For details, see [user management tutorial](#TODO).
+* [Role](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model#standard-roles) requirements:
+    * Generating the SCIM API token requires the **Authentication domain manager** role.
+    * Using the **Organization and access** UI to manage group access requires **Authentication domain manager** and **Organization manager** roles.
+
+## Overview [#overview]
+
+Before using this tutorial, we first recommend reading these docs:
+
+* [Where to find user management features in the UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where)
+* [Important user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts)
+* [SCIM API reference](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management/)
+* SCIM 2.0 RFC documents from the Internet Engineering Task Force: [RFC 7642](https://tools.ietf.org/html/rfc7642), [RFC 7643](https://tools.ietf.org/html/rfc7643), [RFC 7644](https://tools.ietf.org/html/rfc7644)
+
+This tutorial will walk you through how to:
+
+* [Configure your Authentication Domain for SCIM](#configure-scim)
+* [Create users and user groups in your system](#create-in-idp)
+* [Connect to the SCIM API](#connect)
+* [Create users in your Authentication Domain](#create-users)
+* [Create groups in your Authentication Domain](#create-groups)
+* [View users and groups in your Authentication Domain](#view)
+* [Update a user's attributes](#update-user)
+* [Update a group's members](#update-group)
+* [Delete users and groups](#delete)
+* [Manage user type with SCIM](#scim-user-type)
+
+## Configure your Authentication Domain for SCIM [#configure-scim]
+
+Before you can use the SCIM API, you must first [enable SCIM for your authentication domain](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#source-users). Note the API access token will be shown only once after saving the configuration, so keep it someplace safe! 
+
+<Callout variant="tip">
+If you need to view a bearer token later, the only way to do this is to generate a new one, and that will invalidate the old one and any integrations using the old token.
+</Callout>
+
+## Create users and user groups in your system [#create-in-idp]
+
+The SCIM API is typically used for importing users and groups into New Relic from a third-party identity provider. Create users and groups in your system before proceeding in this tutorial.
```

Comment:
```suggestion
The SCIM API is typically used by scripts for importing users and groups into New Relic from a database or a third-party identity provider that doesn't have pre-configured configs for New Relic. 
```

Reply:
When I read this it wasn't totally clear to me whether this was meant to support weirdo systems via curl or if SCIM was just a thing that comes for free when you do something like Auth0. I _think_ after reading this that it's more for the weirdo systems, so I added some words to try to emphasize that. 

I believe the tutorial would work even if you didn't have anything in your system, because the tutorial is just curls. So Removed that second sentence.